taqueria-plugin-taquito/common.ts

```diff
@@ -0,0 +1,200 @@
+import {
+	getAccountPrivateKey,
+	getDefaultAccount,
+	getNetworkConfig,
+	getSandboxAccountConfig,
+	getSandboxAccountNames,
+	getSandboxConfig,
+	sendAsyncErr,
+	sendErr,
+	TAQ_ROOT_ACCOUNT,
+} from '@taqueria/node-sdk';
+import { Environment, NetworkConfig, RequestArgs, SandboxAccountConfig, SandboxConfig } from '@taqueria/node-sdk/types';
+import { importKey, InMemorySigner } from '@taquito/signer';
+import { TezosToolkit } from '@taquito/taquito';
+
+export interface OriginateOpts extends RequestArgs.ProxyRequestArgs {
+	contract: string;
+	storage: string;
+	alias?: string;
+	sender?: string;
+}
+
+export interface TransferOpts extends RequestArgs.ProxyRequestArgs {
+	contract: string;
+	tez?: string;
+	param?: string;
+	entrypoint?: string;
+	sender?: string;
+}
+
+export interface InstantiateAccountOpts extends RequestArgs.ProxyRequestArgs {
+}
+
+export interface FundOpts extends RequestArgs.ProxyRequestArgs {
+}
+
+// To be used for the main entrypoint of the plugin
+export type IntersectionOpts = OriginateOpts & TransferOpts & InstantiateAccountOpts & FundOpts;
+
+// To be used for common functions in this file
+type UnionOpts = OriginateOpts | TransferOpts | InstantiateAccountOpts | FundOpts;
+
+export const getFirstAccountAlias = (sandboxName: string, opts: UnionOpts) => {
```

Comment:
We seem to be mixing terms here. We have an "account alias" and an "account name". I think we should just use "account names".

Reply:
As such, I'd suggest renaming this function to `getFirstAccountName`. However, the term account is something ambiguous as well given that we have declared accounts and account instances.

So perhaps we should have `getSandboxAccounts` and `getFirstSandboxAccount`. To get the account name, it would be nice if we could do `getFirstSandboxAccount(sandbox).name`.